### ISSUE TITLE
Could not invoke RNSpotify.playURI, dlopened failed

### ISSUE THREAD

**Issue by AntonMyr** [lufinkey/react-native-spotify#130]
**React-native version: **0.59.9
**Android version: **9

I tried the example app and everything went well until I came to the `Spotify.playURI` part because then I got this error:
[
![IMG_20190612_131804](https://user-images.githubusercontent.com/16903933/59347509-1ead1500-8d15-11e9-9547-e96bbea5c1c4.jpg)

Error message:
`dlopen failed: library "/vendor/lib64/libgnustl_shared.so" needed or dlopened by "/system/lib64/libnativeloader.so" is not accessible for the namespace "classloader-namespace"`

**Comment by AntonMyr** [comment]
I also tried replacing playURI with `queueURI()` and `skipToNext()` and got the same error.

**Comment by lufinkey** [comment]
What device is this running on?

**Comment by AntonMyr** [comment]
It's running on Huawei p20 pro

**Comment by lufinkey** [comment]
Is that 64 bit?

**Comment by AntonMyr** [comment]
Yes it is.

**Comment by IbrahimCanKALYA** [comment]
@AntonMyr did u find any solution? If u find could share it ?

**Comment by AntonMyr** [comment]
@IbrahimCanKALYA No, unfortunately not. I'm not working on this project anymore.

**Comment by IbrahimCanKALYA** [comment]
@AntonMyr thanks for reply.

**Comment by IbrahimCanKALYA** [comment]
If you are building 64 bit apk with new feature of react-native & requirement of play store.

you should remove packaging from build.gradle 

packagingOptions {
    pickFirst 'lib/armeabi-v7a/libgnustl_shared.so'
    pickFirst 'lib/x86/libgnustl_shared.so'
    exclude 'lib/arm64-v8a/libgnustl_shared.so'
    exclude 'lib/x86_64/libgnustl_shared.so'
}

removing packaging options solved problem

**Comment by iamcxa** [comment]
> If you are building 64 bit apk with new feature of react-native & requirement of play store.
> 
> you should remove packagingOptions from build.gradle
> 
> ```
> packagingOptions {
>     pickFirst 'lib/armeabi-v7a/libgnustl_shared.so'
>     pickFirst 'lib/x86/libgnustl_shared.so'
>     exclude 'lib/arm64-v8a/libgnustl_shared.so'
>     exclude 'lib/x86_64/libgnustl_shared.so'
> }
> ```
> 
> removing packaging options solved problem

I can confirm after removing packagingOptions below, the playURI method works again.

**Comment by chalupagrande** [comment]
Can also confirm removing packagingOptions in `/android/app/build.gradle` solved the problem. 
For others, this _also_ fixed a problem I was having when logging in (with facebook) the app would crash and require reopening. After reopening, my app would initialize with my credentials tho. 
Thanks and hope this helps!
